src/libnnstreamer-edge/nnstreamer-edge-internal.c

```diff
@@ -72,6 +74,8 @@ typedef struct
 
   /* MQTT or AITT handle */
   void *broker_h;
+  void *custom_lib;
+  void *custom_instance;
```

Comment:
suggestion: adding new internal struct for custom connection
```
typedef struct 
{
  void *dlhandle;
  void *instance;
  void *priv;
} custom_connection_s;

/* Struct for custom connection */
custom_connection_s custom;
```

Reply:
Updated. Thanks!